criação dos novos botões, para dev.

brunowe opened the pull request.

brunowe committed:
criei outro campo input e botão com as devidas normalizações

amaralc reviewed (changes requested):
Sugestão de bruno: trocar texto por placeholder

amaralc committed:
code: adicionei placeholder

Por enquanto ainda não irá aparecer pois o código já o substitui imediatamente por outra mensagem.
